src/rocdata.jl

```diff
@@ -1,16 +1,17 @@
 ## Data Preparation for ROC Analysis
 
 struct _PreparedROCData{T<:Real}
-	scores::Vector{T}
-	labels::BitVector
+	scores::AbstractVector{T}
+	labels::AbstractVector{Bool}
```

Comment:
I think the changes to the type of `scores` isn't important here so that can stay as `Vector`, but labels being a `BitVector` breaks if you input `[false,true,false]` as that's a `Vector{Bool}` which isn't a `BitVector`

Reply:
We can go for `_PreparedROCData{T<:Real, B<:AbstractVector{Bool}}` then.

Reply:
`Union{Vector{Bool}, BitVector}` instead of `AbstractVector{Bool}` could be also an option.